core/src/main/resources/reference.conf

```diff
@@ -1,4 +1,9 @@
 aetr {
+  meta = {
+    host = "http://localhost"
+    complete_callback_path = "/api/v1/complete/:token"
```

Comment:
Smart, this lets people put a reverse proxy and they can have the endpoint be whatever they want

Reply:
🎊 🔥